graphql/typeDefs.ts

```diff
@@ -217,6 +217,7 @@ export default gql`
     users: [User]
     currentUser: User
     chatUrl: String
+    modules: [Module!]
```

Comment:
Is there a reason why we want to make `modules` undefined? Would `modules: [Module!]!` be better so we don't have to deal with undefined?

Reply:
None of the lessons have `modules` or `exercises`. While working on DOJO pages, some pages might need to query the lesson modules. It'll throw an error if it's `[Module!]!`.